easybuild/easyconfigs/s/SlurmViewer/SlurmViewer-1.0.2-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,61 @@
+# #
+# Authors:    
+#  - Patrick de Koning <p.j.h.de_koning@lumc.nl>
+#  - Prerak Mody <prerakmody@gmail.com>
+# #
+easyblock = 'PythonBundle'
+
+name = 'SlurmViewer'
+version = '1.0.2'
+
+homepage = 'https://gitlab.com/lkeb/slurm_viewer'
+description = """One-stop command to view the status of a Slurm cluster, including nodes, jobs and resource utilization"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
```

Comment:
Because you've bumped the toolchain to GCC 13.3.0 here, you will also have to bump the versions of the dependencies. Not sure if we have all of them for this toolchain, let me check...